src/main/java/minecraftbyexample/ClientOnlyProxy.java

```diff
@@ -103,7 +105,7 @@ public void postInit()
     minecraftbyexample.mbe11_item_variants.StartupClientOnly.postInitClientOnly();
     minecraftbyexample.mbe12_item_nbt_animate.StartupClientOnly.postInitClientOnly();
     minecraftbyexample.mbe13_item_tools.StartupClientOnly.postInitClientOnly();
-    minecraftbyexample.mbe15_item_dynamic_item_model.StartupClientOnly.postInitClientOnly();
```

Comment:
Was this deletion intentional?

Reply:
I guess so 😂 (I see the additional deletions below)

Reply:
Now I'm not so sure. Going through and cleaning up this PR, I'm going to try putting this back in

Reply:
Huh? It's still there in the source...

Reply:
...never mind, I was looking at the wrong section